scripts/gdb.py

```diff
@@ -43,6 +44,20 @@ def find_vma_base(addr):
         return start-offset
   return None
 
+def get_text_offset(file_name):
+  #command=['readelf', '-SW', file_name]
```

Comment:
Can you not make it work with an array? It would be better to avoid string parsing by the shell

Reply:
Thanks. That works too. Was not sure which one would be more preferable.
Is the non shell mode better to avoid shell interpreting stuff like "*" ?

Reply:
Exactly